simulation_overview.md

```diff
@@ -0,0 +1,72 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.9.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(sec_simulation_overview)=
+
+# Tree sequences and simulation
+
+Simulations are important in population genetics for many reasons:
```

Comment:
At some point in the future, it would be cool to add some links to papers that have used `msprime` for each of these purposes.

Reply:
Nice idea. I'll put it in a comment